MekHQ/src/mekhq/campaign/CampaignOptions.java

```diff
@@ -257,7 +257,8 @@
     private int adminXPPeriod;
     private int edgeCost;
 
-    //repair related
+    //region Repair and Maintenance Tab
```

Comment:
Can we roll back all but the actual addition of `logMaintenance` ? Order of XML elements is significant, while perhaps we don't define it as such, it impacts XML diff tools' semantic analysis.

Reply:
I'll roll back that part, but we should probably think about developing a final syntax order instead of the jumble we currently have.

Reply:
Rolled back the XML Write changes